vividus-tests/src/main/resources/properties/environment/system/saucelabs/mobile_app/ios/environment.properties

```diff
@@ -9,4 +9,7 @@ bdd.variables.global.element-to-ignore=xpath(//XCUIElementTypeOther[./XCUIElemen
 bdd.variables.global.browser-app=com.apple.mobilesafari
 bdd.variables.global.main-app=org.reactjs.native.example.vividusTestApp
 
+bdd.variables.global.device-name-low-dpr=iPhone 7 Simulator
+bdd.variables.global.device-name-high-dpr=iPhone 11 Pro Max Simulator
```

Comment:
I think these variables can be inlined

Reply:
Do I need to replace`${target-platform}-high-dpr` to `ios-high-dpr` and `${target-platform}-low-dpr` to `ios-low-dpr`?

Reply:
done